main.py

```diff
@@ -247,15 +276,15 @@ async def main():
     #     request_rapidapi_indeed_data(), request_linkedin_data()
     # )
     # filtered_job_opps = utils.gpt_job_analyze(job_opps)
-    # opps.ingest_opportunities(filtered_job_opps, "jobs_table")
+    # opps.ingest_opportunities(filtered_job_opps)
 
     # # Consolidates all job-related opportunities into a comprehensive List[Opportunity], eliminating repetitive calls to the LLM SERVER.
     # internship_opps = utils.merge_all_opportunity_data(
-    #     request_github_internship24_data(),
     #     request_linkedin_internship24_data(),
+    #     request_github_internship24_data(),
     # )
     # filtered_internship_opps = utils.gpt_job_analyze(internship_opps)
-    # opps.ingest_opportunities(filtered_internship_opps, "internship_table")
+    # opps.ingest_opportunities(filtered_internship_opps)
```

Comment:
This seems to be commented out to avoid accidentally trigger the webhook on a dev branch. Can we instead change the workflow to [only trigger on main?](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters)

Reply:
Implemented [here](https://github.com/acmcsufoss/acmOpportunities/pull/11/commits/1084fabdbf39ec53e031f7dd4998f397ca913f72)